core/src/uk/ac/york/student/game/activities/Streak.java

```diff
@@ -0,0 +1,58 @@
+package uk.ac.york.student.game.activities;
+
+import lombok.Getter;
+
+/**
+ * Class which represents a streak of activities.
+ */
+@Getter
+public class Streak {
+    /**
+     * A Boolean value used to lock the value of streak for if the player does the activity twice in one day.
+     */
+    Boolean daily;
```

Comment:
Unless this is intended to be nullable you should probably specify the type as `boolean` not `Boolean`

Reply:
Fixed